docs/source/markdown/podman-secret-ls.1.md.in

```diff
@@ -28,6 +28,20 @@ Valid filters are listed below:
 
 Format secret output using Go template.
 
+Valid placeholders for the Go template are listed below:
+
+| **Placeholder**     | **Description**                     |
+| ------------------- | ----------------------------------- |
```

Comment:
hmm, I thought this divider would have to line up with the below, but it seems to work OK as is.  It would be nice to keep them consistent though.